src/core/scope.c

```diff
@@ -51,6 +52,21 @@ static void scope_done(Unit *u) {
         s->timer_event_source = sd_event_source_disable_unref(s->timer_event_source);
 }
 
+static int scope_running_timeout(Scope *s) {
+        usec_t delta = 0;
+
+        assert(s);
+
+        if (s->runtime_rand_extra_usec) {
```

Comment:
as per coding style: for arithmetic types (such as integers) please do explicit non-zero comparisons. We only use C's downgrade to bool feature for things that are actual bools or boolean-like, and as the one special exception for pointers. But for numeric types, please compare explicitly with `!= 0`